platform/darwin/src/http_file_source.mm

```diff
@@ -202,6 +203,7 @@ void cancel() {
 
     @autoreleasepool {
         NSURL* url = [NSURL URLWithString:@(resource.url.c_str())];
+        MGLLogDebug(@"Requesting URI: %@", url.relativePath);
```

Comment:
Nit: if this is HTTPFileSource, are we always requesting via URL (and not the more generic URI)?

Reply:
We request the URL but it contains private information, that's why I'm logging the URI only. Since this was meant to be public had that restriction.